### PR TITLE
Course view

### PR DESCRIPTION
**Pull request by igottheim**
Added initial folder/file setup for front end for course information/lessons

### CODE REVIEW

**Comment by umuthopeyildirim** [comment]
Where is the preview?

File: src/views/Course.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import CoursesHero from "../components/Courses/CoursesHero";
```
Comment (umuthopeyildirim) [inline src/views/Course.jsx:1]: You are using /Courses instead of /Course

File: src/views/Course.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import CourseTextArea from "../components/Course/CourseTextArea";
+import CoreBar from "../components/CoreBar";
+import CoreFooter from "../components/CoreFooter";
+
+function Course() {
```
Comment (umuthopeyildirim) [inline src/views/Course.jsx:6]: You need to recieve props here from main.js

File: src/views/Course.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <div>
+   <CoreBar/>
+   <CoursesHero/>
```
Comment (umuthopeyildirim) [inline src/views/Course.jsx:10]: After reciving props you need to send those probs to children's.

File: src/main.jsx [hunk 4]
```diff
@@ -18,6 +19,10 @@ ReactDOM.createRoot(document.getElementById('root')).render(
         {/* Landing Pages */}
         <Route path="/" element={<Landing />} />
         <Route path="/courses" element={<Courses />} />
+        <Route path="/course/software-engineering" element={<Course/>}/>
```
Comment (umuthopeyildirim) [inline src/main.jsx:22]: Not like this it should be like this

\<Route path="/course/:course" element={\<Course course=course />}/>

[https://reactrouter.com/en/main/hooks/use-params](https://reactrouter.com/en/main/hooks/use-params) check this out

File: src/components/Course/CourseTextArea.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { Hero , Textarea} from "react-daisyui";
+
+
+
+function CourseTextArea() {
+  return(
+    <Textarea>
```
Comment (umuthopeyildirim) [inline src/components/Course/CourseTextArea.jsx:7]: Dont get it lesson markdown or lesson navbar?

File: src/components/Course/CourseHero.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        <p className="py-6">
+              Course HEROOOOO      
+        </p>
+      </div>
+    </Hero.Content>
+  </Hero>
+  )
+}
+
+export default CourseHero;
```
Comment (umuthopeyildirim) [inline src/components/Course/CourseHero.jsx:21]: Not seeing this in preview

**Review by umuthopeyildirim — CHANGES_REQUESTED** [review]
Please see the comments.

**Comment by umuthopeyildirim** [comment]
Preview adress  
https://b0ced469.flatironopensource.pages.dev/course/software-engineering

File: src/main.jsx (hunk 4 above)
Comment (umuthopeyildirim) [reply src/main.jsx:22]: https://refine.dev/blog/react-router-useparams/

File: src/views/Course.jsx (hunk 1 above)
Comment (igottheim) [reply src/views/Course.jsx:1]: Fixed this

File: src/views/Course.jsx (hunk 2 above)
Comment (igottheim) [reply src/views/Course.jsx:6]: Will need to figure out how to pull props

File: src/main.jsx (hunk 4 above)
Comment (igottheim) [reply src/main.jsx:22]: fixed this

File: src/components/Course/CourseTextArea.jsx (hunk 5 above)
Comment (igottheim) [reply src/components/Course/CourseTextArea.jsx:7]: this was just for a template.

Comment (igottheim) [reply src/components/Course/CourseTextArea.jsx:7]: to determine how we will input everything from the class data

File: src/components/Course/CourseHero.jsx (hunk 6 above)
Comment (igottheim) [reply src/components/Course/CourseHero.jsx:21]: Should be fixed

File: src/components/Course/CourseTextArea.jsx (hunk 5 above)
Comment (umuthopeyildirim) [reply src/components/Course/CourseTextArea.jsx:7]: Okey lets just keep it for now

File: src/views/Course.jsx (hunk 2 above)
Comment (umuthopeyildirim) [reply src/views/Course.jsx:6]: You need to import useParams _from_ react-router-dom

File: src/components/Course/CourseTextArea.jsx (hunk 5 above)
Comment (umuthopeyildirim) [reply src/components/Course/CourseTextArea.jsx:7]: Agreed?

File: src/views/Course.jsx (hunk 2 above)
Comment (igottheim) [reply src/views/Course.jsx:6]: I figured this out, will push shortly

File: src/views/Course.jsx (hunk 3 above)
Comment (igottheim) [reply src/views/Course.jsx:10]: figured this out, will need to make a more extensive object but i have the information for the specific course going to the children!

File: src/components/Course/CourseTextArea.jsx (hunk 5 above)
Comment (igottheim) [reply src/components/Course/CourseTextArea.jsx:7]: agreed

**Review by umuthopeyildirim — APPROVED** [review]
LGTM!

**Pull request merged**